pkg/device/device-util.go

```diff
@@ -341,7 +358,7 @@ func GetPartitionList(diskName string, diskMetaName string, free bool) ([][]stri
 			continue
 		}
 		if tmp[0] == "Partition" && tmp[1] == "Table" && tmp[2] != "gpt" {
-			klog.Infof("Disk: %s Not an GPT Partitioned Disk", diskName)
```

Comment:
In line 339 (old) 356 (new) we need to split the line based ":" delimiter

Reply:
-m, --machine
displays machine parseable output
Use this option to get the out put in delimited way.

Reply:
Done.